netaddr.go

```diff
@@ -727,8 +727,12 @@ func (p IPPrefix) IPNet() *net.IPNet {
 //
 // An IPv4 address will not match an IPv6 prefix.
 // A 4-in-6 IP will not match an IPv4 prefix.
+// A zero-value IP will not match any prefix.
```

Comment:
Actually, I already merged this, but reading this doc again, it could be read as meaning p.IP can also be zero to make this return false.

Currently a zero p.IP will still panic.

Reply:
I thought it was clear from the IP <-> prefix separation also found in lines above ("X IP will not match Y prefix" <-> "X `addr` will not match Y `p`"). But maybe a zero `p.IP` should return false for all IPs too?